deepmd/train/run_options.py

```diff
@@ -174,14 +123,15 @@ def print_resource_summary(self):
         log.info("---Summary of the training---------------------------------------")
         if self.is_distrib:
             log.info("distributed")
-            log.info(f"world size:              {self.world_size}")
+            log.info(f"world size:           {self.world_size}")
             log.info(f"my rank:              {self.my_rank}")
-            log.info(f"node list:          {self.nodelist}")
+            log.info(f"node list:            {self.nodelist}")
         log.info(f"running on:           {self.nodename}")
-        if self.gpus is None:
-            log.info(f"CUDA_VISIBLE_DEVICES: unset")
-        else:
-            log.info(f"CUDA_VISIBLE_DEVICES: {self.gpus}")
+        log.info(f"computing device:     {self.my_device}")
+        if tf.test.is_built_with_gpu_support():
```

Comment:
`is_built_with_gpu_support` is added in https://github.com/tensorflow/tensorflow/commit/c9e11b03d14edcfd1635d7e7593e3fb3ce59615b since v1.15.0.

@amcadmus Do we have plan to deprecated TensorFlow<=1.14?

Reply:
Since we declared ROCM support in DeePMD-Kit 2.0, TensorFlow v1.14 should be ignored.

Reply:
If we abandon TensorFlow v1.14 support, changes of CI environment need to involve. This may be a huge effort, thus let's delay it to another pull request or discussion. @njzjz @amcadmus

Reply:
I suggest we should support both TF <= 1.14 and > 1.14. 

Unless we need any **critical** feature only supported in higher versions of TF, we keep supporting older versions of TF.